core/src/main/java/com/distkv/dst/core/struct/slist/SortedListLinkedImpl.java

```diff
@@ -297,4 +285,22 @@ private int compares(
     }
   }
 
+  /**
+   * Check whether the list value has duplicated members when initially put.
+   *
+   * @param sortedListEntities The list value
+   * @return True if the list value has duplicated members, otherwise return false.
+   */
+  private boolean hasDuplicatedMembers(
+      List<SortedListEntity> sortedListEntities) {
+    final Set<SortedListEntity> sortedListEntitySet = new HashSet<>();
+    for (final SortedListEntity e : sortedListEntities) {
+      if (sortedListEntitySet.contains(e)) {
+        return true;
+      }
+      sortedListEntitySet.add(e);
```

Comment:
I think the cost it too expensive to add all items to a set to check if there're duplicated members.

Do you have any better way? like using their own storage.